### ISSUE TITLE
Bot Stopped Working

### ISSUE THREAD

**Issue by grumpybobs** [div0ky/fsb_idle#16]
I turned the bot on last night before bed. 
I woke up to it on this screen (pictured)
I believe it stopped working early on in the night/shortly after being turned on, because I did not have much progress to note, when I woke.
![error]
(https://user-images.githubusercontent.com/60832680/74107683-58e6e480-4b38-11ea-94ff-5bb8bcc005a8.PNG)

I believe the error has something to do with Guild Expeditions. This is the second time it stopped working on that screen. (first time I thought was user error. I no longer believe it was user error.)

**Comment by div0ky** [comment]
Huh. Mine was stopped on exactly the same screen. I don’t suppose that’s coincidence. I need to add better logging because, until then, I don’t know what caused this bug.

**Comment by div0ky** [comment]
>I turned the bot on last night before bed.  ...

@grumpybobs it’s stopped **twice** on that screen for you now?

**Comment by grumpybobs** [comment]
You know...I just read your comment on discord regarding the itunes popup.
I had Groupy, and one of the other stardock apps, down in my taskbar tray, saying I needed to update them.
They did not pop up over firestone, but they did pop up in the middle of the night....

**Comment by grumpybobs** [comment]
Although....myabe they did pop up over it, and got "auto clicked away" but still caused an issue. IDK

**Comment by div0ky** [comment]
Hmm. Discovered mine not working again on yet again the same screen. It must be something with the Guild Expeditions as you suggested. I'll look into it.

**Comment by grumpybobs** [comment]
Bot stopped about an hour after starting it, at expeditions.

**Comment by grumpybobs** [comment]
Stopped after about an hour, on expeditions.

**Comment by div0ky** [comment]
Did you mean to close this issue? Your comment makes me think that was accidental.

**Comment by grumpybobs** [comment]
accidental

**Comment by grumpybobs** [comment]
Started bot, and left house at 4:45pm. 
Returned home at 6pm and bot had stoppped

Restarting bot

**Comment by grumpybobs** [comment]
I keep restarting it. Sometimes it lasts for an hour or two, but it just stopped after 4 minutes.

**Comment by grumpybobs** [comment]
Possible total fluke...but I just had the bot, 2 times in a row, last about 30 seconds before stopping at Guild Expeditions.

Those times I had turned on auto leader special moves.
I turned it back off, and the bot did not stop at expeditions.

Again, maybe just a fluke, as I have seen it on before, without even turning it, and the bot worked for longer than 30 seconds, but...thought it might be worth noting, nonetheless.

I'll keep it off and see how long it last this time.
